docs/index.md

```diff
@@ -0,0 +1,93 @@
+# Ops documentation
+
+```{toctree}
+:maxdepth: 2
+:hidden: true
+
+tutorial/index
+howto/index
+reference/index
+explanation/index
+```
+
+
+
+The Ops library (`ops`) is a Python framework for writing and testing Juju charms.
+
+> [See it on PyPI](https://pypi.org/project/ops/)
+
+The library provides:
+
+- {ref}`ops_main_entry_point`, used to initialise and run your charm
+- {ref}`ops`, the API to respond to Juju events and manage the application
+- {ref}`ops_pebble`, the Pebble client, a low-level API for Kubernetes containers
+- {ref}`ops_testing`, the recommended API for unit testing charms
+- {ref}`ops_testing_harness`, the deprecated API for unit testing charms
+
+You can structure your charm however you like, but with the `ops` library, you get a framework that promotes consistency and readability by following best practices. It also helps you organise your code better by separating different aspects of the charm, such as managing the application's state, handling integrations with other services, and making the charm easier to test.
+
+
+---------
+
+## In this documentation
+
+````{grid} 1 1 2 2
+
+```{grid-item-card} [Tutorial](tutorial/index)
+:link: tutorial/index
+:link-type: doc
+
+**Start here**: a hands-on introduction to Example Product for new users
```

Comment:
Not sure how I missed this before :laughing:

```suggestion
**Start here**: a hands-on introduction to `ops` for new users
```

Reply:
:sweat_smile: I think we should fix this before merging, if possible

As a follow-up action for later, we can look at making the "In this documentation" section more relevant to Ops. Maybe add some featured links, like we do for Pebble?